martha/martha_v3.js

```diff
@@ -0,0 +1,68 @@
+const {dataObjectUriToHttps, parseRequest} = require('../common/helpers');
+const {bondBaseUrl, determineBondProvider, BondProviders} = require('../common/bond');
+const apiAdapter = require('../common/api_adapter');
+
+
+async function maybeTalkToBond(req, provider = BondProviders.default) {
+    // Currently HCA data access does not require additional credentials.
+    // The HCA checkout buckets allow object read access for GROUP_All_Users@firecloud.org.
+    if (req && req.headers && req.headers.authorization && provider !== BondProviders.HCA) {
```

Comment:
Needs moar conditionals-- JDR requests are not supposed to contact bond.

Reply:
Yes. This is part of [WA-179](https://broadworkbench.atlassian.net/browse/WA-179).